sr/robot3/robot.py

```diff
@@ -59,6 +60,8 @@ def __init__(
             Robot under. See :ref:`Environments` for more information.
         :param ignored_ruggeduinos: List of Ruggeduino serial numbers to ignore.
             See :ref:`Custom Ruggeduino Firmware` for more information.
+        :param legacy_camera_axis: Use the coordinate systems the camera used
+            prior to SR OS 2023.2.0.
```

Comment:
```suggestion
            prior to SR OS 2023.1.0.
```

The version we're releasing this week is 2023.1.0. Unless you meant the next version afterwards?